iroha/src/peer.rs

```diff
@@ -1,91 +1,92 @@
-use crate::isi::Command;
+use crate::isi::Contract;
 use parity_scale_codec::{Decode, Encode};
 
-/// The purpose of add peer command is to write into ledger the fact of peer addition into the
-/// peer network. After a transaction with AddPeer has been committed, consensus and
-/// synchronization components will start using it.
 #[derive(Clone, Debug, PartialEq, Encode, Decode)]
-pub struct AddPeer {
-    pub peer: Peer,
+pub struct Peer {
+    pub address: String,
+    pub peer_key: [u8; 32],
```

Comment:
Maybe use type alias for `[u8; 32]` here?

Reply:
Please create a special type for the public keys. It could be just alias for this array. In future we will replace public key with Multihash type (to support different kind of cryptos from Ursa). This will be done much easier if we just update public key alias by the new type